lib/playground.dart

```diff
@@ -343,21 +349,20 @@ class Playground implements GistContainer {
   List<Element> _getTabElements(Element element) =>
       element.querySelectorAll('a');
 
-  void _toggleDocTab() {
-    ga.sendEvent('view', 'dartdoc');
-    docHandler.generateDoc(_docPanel);
+  void _toggleResultTab() {
+    ga.sendEvent('view', 'result');
     // TODO:(devoncarew): We need a tab component (in lib/elements.dart).
     querySelector('#output').style.display = "none";
     querySelector("#consoletab").attributes.remove('selected');
 
-    _docPanel.style.display = "block";
-    _docTab.setAttribute('selected','');
+    _frame.style.display = "block";
+    _resultTab.setAttribute('selected','');
   }
 
   void _toggleConsoleTab() {
```

Comment:
Rather than toggle, I'd prefer these to be explicit "ShowConsoleTab" "HideConsoleTab" commands. That way they don't depend on the previous state.